schema.sql

```diff
@@ -45,3 +45,40 @@ ALTER TABLE animals
 ADD CONSTRAINT Owners_ID 
 FOREIGN KEY (owner_id) 
 REFERENCES owners (id);
+
+DAY 4,
```

Comment:
- [ ] This `DAY 4` would you remove or make a comment it, because it compromise the execution

Reply:
Alright, thanks.

👍

Reply:
:+1: This is addressed successfully :heavy_check_mark: